schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    name  varchar(100),          
+    date_of_birth  date,           
+    escape_attempts int,          
+    neutered  boolean,
+    weight_kg real
```

Comment:
- [ ] I would recommend including the `id` column in the `animals` table.

[OPTIONAL]

- You can mark the `id` column as primary key.

Reply:
@Dipeshtwis 
It is done.